idunn/utils/categories.yml

```diff
@@ -1,144 +1,602 @@
+        raw_filters:
+          - "camp_site,*"
+          - "caravan_site,*"
+        pj_what: "camping"
+
+    station:
+        regex: "^gare( |$)"
+        raw_filters:
+          - "railway,station"
+          - "railway,tram_stop"
+          - "railway,halt"
+        pj_what: "gare"
+
+    shop_butcher:
+        raw_filters:
+          - "*,butcher"
+        pj_what: "boucher"
+
+    attraction:
+        raw_filters:
+          - "nightclub,*"
+          - "leisure,escape_game"
+          - "leisure,golf_course"
+          - "leisure,miniature_golf"
+          - "tourism,aquarium"
+          - "tourism,artwork"
+          - "tourism,attraction"
+          - "tourism,theme_park"
+          - "tourism,zoo"
+        pj_what: "loisir"
+
+    health_hospital:
+        raw_filters:
+          - "clinic,*"
+          - "hospital,*"
+        pj_what: "hôpital"
         pj_filters:
-            - "Hôpital"
+          - "Hôpital"
+
+    health_doctors:
         raw_filters:
-            - "clinic,*"
-            - "hospital,*"
-        regex: "health|sante|hopital|hospital|clinic|clinique"
-    service:
-        pj_what: "poste"
+          - "doctors,*"
+          - "doctor,*"
+        pj_what: "médecin généraliste"
+
+    health_dentist:
+        raw_filters:
+          - "dentist,*"
+        pj_what: "dentiste"
+
+    health_physiotherapist:
+        raw_filters:
+          - "physiotherapist,*"
+        pj_what: "kinésithérapeute"
+
+    health_pharmacy:
+        raw_filters:
+          - "pharmacy,*"
+        pj_what: "pharmacie"
         pj_filters:
-            - "envoi, distribution de courrier, de colis"
+          - "Pharmacie"
+
+    health_psychotherapist:
         raw_filters:
-            - "post_office,*"
-        regex: "(^| )post"
+          - "psychotherapist,*"
+        pj_what: "psychologue"
+
+    health_other:
+        raw_filters:
+          - "optometrist,*"
+          - "alternative,*"
+          - "audiologist,*"
+          - "birthing_center,*"
+          - "blood_bank,*"
+          - "blood_donation,*"
+          - "centre,*"
+          - "counselling,*"
+          - "dialysis,*"
+          - "laboratory,*"
+          - "midwife,*"
+          - "nurse,*"
+          - "occupational_therapist,*"
+          - "podiatrist,*"
+          - "rehabilitation,*"
+          - "speech_therapist,*"
+        pj_what: "santé"
+
+    library:
+        raw_filters:
+          - "library,*"
+        pj_what: "bibliothèque"
+
     police:
+        regex: "(^| )(police|gendar)"
+        raw_filters:
+          - "police,*"
         pj_what: "police"
         pj_filters:
-            - "services de gendarmerie, de police"
+          - "services de gendarmerie, de police"
+
+    shop_optician:
         raw_filters:
-            - "police,*"
-        regex: "(^| )(police|gendar)"
-    fitness:
-        pj_what: "sport"
+          - "*,optician"
+        pj_what: "opticien"
+
+    grave_yard:
+        raw_filters:
+          - "grave_yard,*"
```

Comment:
A class "cemetery" exists and should be used instead:
https://github.com/Qwant/openmaptiles/blob/master/layers/poi/class.sql#L55